spec/spec_helper.rb

```diff
@@ -13,15 +13,13 @@
   config.include Helpers
 
   if Jobba::Spec::Utils.use_real_redis?
-    config.before(:suite) do
+    config.before(:each) do
       Jobba.clear_all_jobba_data!
     end
 
-    config.after(:each) do
```

Comment:
I have no idea why, but a couple of specs were failing before this change, complaining that some data already existed (has to do with the change to `clear_all_jobba_data!`)